recipes-ni/nisdbootconfig/nisdbootconfig.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "NI SDboot Config"
+DESCRIPTION = "Bios helper support files"
```

Comment:
Could you make this package description more specific. This recipe isn't generalized to all BIOS support files.

Also NIT: `BIOS` should have every letter capitalized. :)

Reply:
Added the correct description.